sdk-quarkus/pom.xml

```diff
@@ -18,8 +18,13 @@
 
     <dependencies>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>a2a-java-sdk-core</artifactId>
+            <groupId>io.a2a.sdk</groupId>
+            <artifactId>a2a-java-sdk-model</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.a2a.sdk</groupId>
+            <artifactId>a2a-java-sdk-client</artifactId>
```

Comment:
Is the client dependency needed here? I think sdk-quarkus just makes use of constants from the `A2A` class, right? Maybe we could move those constants to `a2a-java-sdk-model`.